docs/manual/hacking.rst

```diff
@@ -10,6 +10,7 @@ install them from your package manager.
 
 * `Nose <http://nose.readthedocs.org/en/latest/>`_
 * `Xephyr <http://www.freedesktop.org/wiki/Software/Xephyr>`_
+* `xrandr <www.x.org/wiki/Projects/XRandR/>`_
 * ``xeyes`` and ``xclock``
 
 On ubuntu, this can be done with ``sudo apt-get install python-nose
```

Comment:
Is `xrandr` included in this `apt-get` command? (I couldn't tell if it was included in `x11-apps` -  I am not familiar with Ubuntu).